src/components/forms/commodity/CommodityForm.tsx

```diff
@@ -112,6 +112,8 @@ async function onSubmit(
   if (action === 'add' || action === 'update') {
     await commodity.save();
   } else if (action === 'delete') {
+    Price.delete({ fk_commodity: commodity });
+    Price.delete({ fk_currency: commodity });
```

Comment:
Delete prices related to that commodity.

Sadly this doesn't invalidate the cache because we've only implemented it for the instance.remove method. Maybe will improve in the future